references/recaptcha-enterprise/sf-recaptcha-enterprise-v1/sharedflowbundle/resources/jsc/JS-AccessReCaptchaEnterprise.js

```diff
@@ -0,0 +1,112 @@
+/**
+  Copyright 2022 Google LLC
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/**
+ * Set date and time.
+ * @return {string}
+ */
+function setDateAndTime() {
+    var today = new Date();
+    return today.toISOString();
+}
+
+// Get reCAPTCHA context variables
+var reCaptchaToken = context.getVariable('flow.recaptcha.token');
+var gcpProjectId = context.getVariable('flow.recaptcha.gcp-projectid');
+var siteKey = context.getVariable('flow.recaptcha.sitekey');
+
+// Set token validity and risk score
+var isTokenValid = false;
+var riskScore = 0;
+
+// Test the token value with format: X-RECAPTCHA-TOKEN-<SCORE>
+switch (reCaptchaToken) {
```

Comment:
switch vs regex and parseFloat?

Reply:
yes using a basic substring and parseFloat is definitively a simpler solution. Thanks